### DISCUSSION TITLE
How to use system proxy in sandboxie with privacy mode ?

### DISCUSSION THREAD

**Discussion by NeonBubbles** [sandboxie-plus/Sandboxie#3652] [Documents and notes]
Due to the restriction in privacy mode, most system resources can not be accessed. 
I use Clash to bypass the internet firewall in China, what should I add in resource access to use system proxy in sandbox(privacy mode)?

**Comment by offhub** [comment]
Please try it with the following setting:

```
NormalKeyPath=HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Internet Settings\Connections\*
```

**Reply by NeonBubbles** [reply]
Thank you!

**Reply by NeonBubbles** [reply]
After several tests, the right setting should use  OpenKeyPath

`OpenKeyPath=HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Internet Settings\Connections\*`
